### DISCUSSION TITLE
Outreachy 2024: Enabling Real-time Citizen Reporting through WhatsApp Integration for Ushahidi

### DISCUSSION THREAD

**Discussion by CeciliaHinga** [ushahidi/platform#4715] [Outreachy and Google Summer of Code]
**Welcome to Ushahidi and the application process for Outreachy 2024!**

We are so excited you are here!!

We have compiled a list of helpful resources and steps to help you make successful Outreachy contributions. Ensure that you do not skip any of the steps below.

**Step 1** - Get familiar with the Ushahidi community:

- Join, say hello, and mention your country in our [community Discord server](https://discord.com/invite/p6BAVx7T9K).  Use the correct channel **_(#outreachy-whatsapp_**) to communicate, help, and get help from the community.
- Read through our Code of Conduct in the #welcome and rules channel on Discord.

**Step 2** - You can find issues to work on through our [[OSS community board](https://github.com/orgs/ushahidi/projects/24/views/1 )]. We have made it easy - check for issue tickets with the Outreachy: WhatsApp 2024 label on them, you will also find the difficulty level on the issue tickets to help you easily identify those that you can tackle based on your experience and/or strengths. 

**Step 3** - Explore the Ushahidi Platform and the different concepts like surveys (also called "forms" in some places) and posts. You can explore by studying and clicking around on the platform's user interface or through our [user documentation](https://docs.ushahidi.com/platform-user-manual/) 

**_Things to Note_**

- You can only work on one issue at a time. We are looking for the quality of the contributions rather than the number of contributions when evaluating the applications.

- Once you are assigned to an issue, we want you to submit a pull request within 2 days, even if it is not yet complete or a working solution. This is to make our contributor's work is "in the open," as is often done in Open Source, and make sure no issues are not being worked on. If you don't show progress within 3 days, we will open it up for other participants. GitHub has a good guide on [how to submit pull requests from a fork](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request-from-a-fork).

- Remember that Ushahidi is an open-source community. The mentors are trying their best to reply to your questions, but there may be some delays. By helping other contributors out in our community Discord server, you are also helping the mentors. We recognize this and see it as something very positive when evaluating your submissions. So, if you see someone asking for help in the chat and know the answer, please help them out! Keep the tone nice and collaborative.

Privacy info/personal info: During the application process, Ushahidi will store your Github handle in a Google sheet. This is done only to make the application process as smooth as possible and to keep track of which issues are being worked on and the progress of each issue. The information stored will be deleted once the application process is over.

Welcome once again. We are looking forward to the awesome contributions you will submit to our community 🌻 🎉 💐

**Comment by egbadon-victor** [comment]
Hello Everyone. I am excited to be here. From reading the above instructions, this link [[OSS community board](https://github.com/orgs/ushahidi/projects/24/views/1)] doesn't seem to work . It takes me to a 404 page

**Comment by siva224513** [comment]
Hello everyone ,I'm siva an outreachy applicant and excited to contributing this project

**Comment by PeaceNaza** [comment]
Hello everyone ,I'm Peace from Lagos Nigeria an outreachy applicant. I'm excite to contributing this project!!

**Comment by princyym** [comment]
hello everyone, I am Princy, a beginner to this project and excited to contribute to this project!
